src/main/java/com/webonise/models/Coordinates.java

```diff
@@ -0,0 +1,50 @@
+package com.webonise.models;
+
+import org.springframework.stereotype.Component;
+
+import java.util.Observable;
+
+@Component
+public class Coordinates extends Observable {
```

Comment:
If you are going to use this class as POJO then remove annotation Component.
Please override hashCode, equals and toString